system/lib/pthread/library_pthread_stub.c

```diff
@@ -288,9 +286,6 @@ int pthread_mutex_destroy(pthread_mutex_t* mutex) { return 0; }
 
 int pthread_mutex_consistent(pthread_mutex_t* mutex) { return 0; }
 
-#define pthread_barrier_t unsigned
```

Comment:
typedef is probably better here no?

How are these not already defined in the header file?

Reply:
They are defined in the header, yeah. For some reason this C file did *not* include the header and just defined things locally as needed... I removed those and included the header, which seems simpler and safer.